qml/Ping360ControlPanel.qml

```diff
@@ -42,13 +33,25 @@ Item {
                 control.to: 100
                 control.onMoved: ping.range = control.value
             }
-            PingSlider {
+
+            PingComboSlider {
+                text: "Receiver Gain: "
```

Comment:
`PingSlider` already adds the ": " for you.
![image](https://user-images.githubusercontent.com/1215497/61895747-c3606c00-aee9-11e9-942e-c86d38c85a21.png)